### ISSUE TITLE
TypeError: loadOAuthConf(...).toPromise is not a function

### ISSUE THREAD

**Issue by BlackYoup** [CleverCloud/clever-tools#406]
I get this error when deploying with the latest beta version.

Here is the full output:
```
~ $ clever deploy -a prod -v      
[DEBUG] Loading app configuration from /home/arnaud/Dev/clevercloud/console3/.clever.json
[DEBUG] Load configuration from environment variables
[DEBUG] Load configuration from /home/arnaud/.config/clever-cloud
[STACKTRACE]
TypeError: loadOAuthConf(...).toPromise is not a function
    at onAuth (/snapshot/clever-tools_2.3.1-beta.0/src/models/git.js:26:40)
    at Function.discover (/snapshot/clever-tools_2.3.1-beta.0/node_modules/isomorphic-git/index.cjs:6287:24)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Object.getRemoteInfo (/snapshot/clever-tools_2.3.1-beta.0/node_modules/isomorphic-git/index.cjs:8964:20)
    at async Object.getRemoteCommit (/snapshot/clever-tools_2.3.1-beta.0/src/models/git.js:60:23)
    at async deployPromise (/snapshot/clever-tools_2.3.1-beta.0/src/commands/deploy.js:20:30) {
  caller: 'git.getRemoteInfo'
}
[/STACKTRACE]
[ERROR] loadOAuthConf(...).toPromise is not a function
```

clever diag:
```
~ $ clever diag
clever-tools  2.3.1-beta.0
Node.js       v12.13.1
Platform      linux
Release       5.6.0+
Architecture  x64
Linux         Exherbo Linux
Packaged      true
Exec path     /usr/x86_64-pc-linux-gnu/bin/clever
User id       user_850dfc46-42e8-4a90-bb82-78c5d72bfbf7
oAuth token   <token>
```

**Comment by hsablonniere** [comment]
Makes sense...

**Comment by hsablonniere** [comment]
I transformed `loadOAuthConf` into an async function in https://github.com/CleverCloud/clever-tools/commit/e5674dcf7b821e3477368d98a68b589c5756b795.

When I rebased the isogit branch to make a release, I forgot to update this code.

**Comment by hsablonniere** [comment]
Fixed in 2.3.1-beta.1

**Comment by BlackYoup** [comment]
Thanks, I confirm the error is now gone.
